snippets/coffee-adapters.cson

```diff
@@ -0,0 +1,28 @@
+'.source.coffee':
+
+  'Ember fixtureAdapter':
+    'prefix': 'ds.fixA'
```

Comment:
Can you use more clear prefix like `fixture-adapter`, so it's similar to the rest of the rest of the project. Also please apply this concept to the rest of the prefixes. `rest-adapter`

Can you also move adapters to `modules.cson`